pkgs/applications/networking/modem-manager-gui/default.nix

```diff
@@ -0,0 +1,50 @@
+{ stdenv, buildEnv, pkgconfig, python3, fetchhg, gtk3, glib, gdbm, gtkspell3, itstool, libappindicator-gtk3, perldevelPackages, glibcLocales, meson, ninja }:
+
+stdenv.mkDerivation rec {
+  name = "modem-manager-gui";
```

Comment:
`name` is supposed to be `modem-manager-gui-${version}`. At least until https://github.com/NixOS/rfcs/pull/35 is implemented.